src/Forms/Checkbox.js

```diff
@@ -30,7 +48,8 @@ const Checkbox = React.forwardRef(({ checked, className, defaultChecked, disable
 
     const classes = classnames(
         className,
-        'fd-checkbox'
+        'fd-checkbox',
+        { [`is-${state}`]: state }
```

Comment:
I want to entertain this thought:  Should we be defining classes with full strings instead of interpolation.

If we use string interpolation we wouldn't work with something like [PurgeCSS](https://purgecss.com/#table-of-contents) in case a consumer wants to configure their project to minimize the imported styles.

So we'd want some utility to add the full strings here
```js
 { ['is-warning']: state.warning, ['is-invalid']: state.invalid, ... }
```

Reply:
I like the idea of this - maybe as a follow up if we want to use purgeCSS in nui-widgets/floorplans. We could go through all the components with these issues.

Reply:
I also prefer to avoid string interpolation for classnames - they don't show up in search and replaces efforts, which inevitably leads to bugs.

Reply:
I can see @bcullman's point and in our legacy applications I completely agree.  However, I will say that in the world of self-contained styles, I don't think this is nearly as big of an issue anymore.

Reply:
Let's 🅿️ this today... the more I think about it, the less I want to make this change